rdock-utils/rdock_utils/common/types.py

```diff
@@ -1,23 +1,38 @@
 from typing import Any
 
-import numpy
+import numpy as np
+import numpy.typing as npt
```

Comment:
I know the standard/recommendation for numpy imports is to give it the `np` alias, but I'd rather have the full name. It's not like it saves a big amount of characters, and it is less "clear" / "pythonic".

This one is a purely stylistic choice, just the same way I'd name a variable `velocity_x`, instead of just `vx`, for anything other than a prototype.